### PR TITLE
Assignment1 api

### PR DESCRIPTION
**Pull request by cschadewitz**
image

### CODE REVIEW

**Comment by NicholasJohnsonDev** [comment]
I will review this

**Comment by NicholasJohnsonDev** [comment]
I will review this

**Comment by NicholasJohnsonDev** [comment]
Compiles and fulfills all requirements.

API project updated to output new welcome message in second pull request: ✔
Screenshot of web browser running the API project included in second pull request: ✔

It would be more clear that the Web and API were different by the screenshot if you gave a different message for each but the port number visible helped clarify that they were different. Pulled down and works just fine.

File: SecretSanta/src/SecretSanta.Web/Startup.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapGet("/", async context =>
                 {
-                    await context.Response.WriteAsync("Hello from Web!");
+                    await context.Response.WriteAsync("Hello from World 2!");
```
Comment (mmwoodfo) [inline SecretSanta/src/SecretSanta.Web/Startup.cs:35]: Shouldn't have the web changes here. No worries about point loss - just a heads up for future projects to stick to instructions. 👍🏻

File: SecretSanta/src/SecretSanta.Api/Startup.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapGet("/", async context =>
                 {
-                    await context.Response.WriteAsync("Hello from API!");
+                    await context.Response.WriteAsync("Hello from World 2!");
```
Comment (mmwoodfo) [inline SecretSanta/src/SecretSanta.Api/Startup.cs:35]: :fire:

**Pull request closed without merging**